python/ribasim/ribasim/delwaq/generate.py

```diff
@@ -73,18 +73,17 @@ def _make_boundary(data, boundary_type):
     """
     bid = _boundary_name(data.node_id.iloc[0], boundary_type)
     piv = (
-        data.pivot_table(
-            index="time", columns="substance", values="concentration", fill_value=-999
-        )
+        data.pivot_table(index="time", columns="substance", values="concentration")
         .reset_index()
         .reset_index(drop=True)
     )
-    # Convert Arrow time to Numpy to avoid needing tzdata somehow
-    piv.time = piv.time.astype("datetime64[ns]").dt.strftime("%Y/%m/%d-%H:%M:%S")
+    piv.time = piv.time.dt.strftime("%Y/%m/%d-%H:%M:%S")
     boundary = {
         "name": bid,
         "substances": list(map(_quote, piv.columns[1:])),
-        "df": piv.to_string(formatters={"time": _quote}, header=False, index=False),
+        "df": piv.to_string(
+            formatters={"time": _quote}, header=False, index=False, na_rep=-999
```

Comment:
Why did you move from the fill_value to na_rep here? Not sure if it matters if the tests run.

Reply:
Just reverting parts of https://github.com/Deltares/Ribasim/pull/1781#discussion_r1773291247. This is slightly nicer that it doesn't write .0 after -999.